redisinsight/desktop/src/lib/cloud/cloud-oauth.handlers.ts

```diff
@@ -37,6 +72,29 @@ export const getTokenCallbackFunction = (webContents: WebContents) => (response:
 export const initCloudOauthHandlers = () => {
   ipcMain.handle(IpcInvokeEvent.cloudOauth, async (event, options: CloudAuthRequestOptions) => {
     try {
+      if (process.env.NODE_ENV === 'development') {
+        const { url } = await sendTcpRequest({
+          action: 'getAuthUrl',
+          options: {
+            sessionMetadata: {
+              sessionId: DEFAULT_SESSION_ID,
+              userId: DEFAULT_USER_ID,
+            },
+            authOptions: {
+              ...options,
+              callback: getTokenCallbackFunction(event.sender),
+            }
+          }
+        })
+
+        await open(url)
+
+        return {
+          status: CloudAuthStatus.Succeed,
```

Comment:
in the CloudAuthStatus enum exists only Success

Reply:
Not really, this is the enum this is using https://github.com/RedisInsight/RedisInsight/blob/be/feature/RI-5876-Fix-start-electron-dev-mode/redisinsight/desktop/src/types/cloud-auth.ts

Reply:
I don't see `Succeed` at this enum

Reply:
my bad, we have several enums with the same name and I have imported the wrong one. Resolved it.